packages/connection-form/.eslintrc.js

```diff
@@ -5,4 +5,35 @@ module.exports = {
     tsconfigRootDir: __dirname,
     project: ['./tsconfig-lint.json'],
   },
+  overrides: [
+    {
+      files: ['./src/**/*.ts', './src/**/*.tsx'],
+      rules: {
+        // 'mongodb' package has been moved to devDependencies in this package, to
+        // aid in the usage of this package inside the VSCode extension's webview.
+        // Otherwise it would have been necessary to provide several polyfills for
+        // the dependencies of 'mongodb' package when webpacking it for an
+        // environment other than node. Hence we are restricting the usage of this
+        // package to type only imports.
+        '@typescript-eslint/no-restricted-imports': [
+          'error',
+          {
+            paths: [
+              {
+                name: 'mongodb',
```

Comment:
I think connection storage was another one that we only want as type-only dep here, right?

Reply:
Thank you, I overlooked this one - added this in [247e5ca](https://github.com/mongodb-js/compass/pull/5207/commits/247e5caf9a92f862aa0f26a65609785444867b14)